pom.xml

```diff
@@ -2206,6 +2208,7 @@
             </repositories>
             <properties>
                 <spark.version>3.4.0-SNAPSHOT</spark.version>
+                <spark.archive.name>spark-${spark.version}-bin-hadoop3.2.tgz</spark.archive.name>
```

Comment:
we don't need it

Reply:
The daily testing does not trigger `externals/kyuubi-download`
```
./build/mvn clean install -Pspark-master -pl externals/kyuubi-spark-sql-engine -am -Dmaven.javadoc.skip=true -V
```
```
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO]
[INFO] Kyuubi Project Parent                                              [pom]
[INFO] Kyuubi Project Common                                              [jar]
[INFO] Kyuubi Project Embedded Zookeeper                                  [jar]
[INFO] Kyuubi Project High Availability                                   [jar]
[INFO] Kyuubi Project Events                                              [jar]
[INFO] Kyuubi Project Hive JDBC Client                                    [jar]
[INFO] Kyuubi Project Hive JDBC Shaded Client                             [jar]
[INFO] Kyuubi Project Engine Spark SQL                                    [jar]
[INFO]
[INFO] ------------------< org.apache.kyuubi:kyuubi-parent >-------------------
```

Reply:
users can use `-Pspark-provided` to skip it, but we can not run kyuubi-server testing either way.